transients_nearby_galaxies/transients_nearby_galaxies_solutions.ipynb

```diff
@@ -0,0 +1,489 @@
+{
```

Comment:
<p>both for this result and in general: how much discussion do we want?</p><p> </p><p>in this example, I think we could make a generic statement to the effect of: "not every transient appears in this FFI, but quite a few of them do"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>I added this: # We can see that a number of transients fall into the field of view of this FFI,</p><p># but we haven't yet looked at if they are timed correctly.</p><p># We can also tell that some of the transients that appear in the corners</p><p># of the field of view won't be caught by our 6.5 degree search radius.</p>